node-require/calculate.js

```diff
@@ -0,0 +1,14 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+if (process.argv[3] === 'plus') {
+  console.log('result: ', add(parseInt(process.argv[2]), parseInt(process.argv[4])));
+} else if (process.argv[3] === 'minus') {
+  console.log('result: ', subtract(parseInt(process.argv[2]), parseInt(process.argv[4])));
+} else if (process.argv[3] === 'times') {
+  console.log('result: ', multiply(parseInt(process.argv[2]), parseInt(process.argv[4])));
+} else if (process.argv[3] === 'over') {
+  console.log('result: ', divide(parseInt(process.argv[2]), parseInt(process.argv[4])));
+}
```

Comment:
To avoid a bunch of repetition it would be best to save `process.argv[2]`, `process.argv[3]`, and `process.argv[4]` into variables. Also when you create the variables you can parse the number so you don't have to do it in 8 different places. For example:
```JAVASCRIPT
const num1 = parseInt(process.argv[2]);
const operator = process.argv[3];
const num2 = parseInt(process.argv[4]);
```

Reply:
That actually sounds better. Laziness comes in here haha